### PR TITLE
Add two new alert levels - Orange and White.

### PR DESCRIPTION
**Pull request by onenineone**
## About the PR

Added two new alert levels with their custom alert sounds to the game and communications console that players can select: orange and white.

The orange alert serves as a middle-ground between blue alert and red alert, focusing on hostile or unknown life forms on board the station. With it's announcement, it states: "There are currently several unknown or hostile life forms aboard the station. All crew members are advised to remain in their departments and to report any aggressive entities to security." while the PDA instructions state: "Crewmembers are advised to remain vigilant and report hostile life forms to security."

While the white alert concentrates on supernatural or otherwise unexplained phenomena which has become more common in the Umbra Sector stations. The announcement states: "There is currently an unexplained abnormality on the station. All crewmembers are to remain vigilant, seek shelter, listen to command orders and to report any unknown phenomena to security or science." whilst the PDA instructions state: "Crewmembers are advised to be remain in their departments and to report any unexplained or unnatural sightings to security or science."

Both of the alerts alter the SO A-2 subsection of the Standard Operating Procedure. For the orange alert, it is:

- Code Orange: Commanding Officer > Head of Security > Executive Officer > Security department > Relevant head-of-department > Relevant department senior > Regular crew

And for the white it's:
- Code White: Commanding Officer > Research Director > Executive Officer > Security department > Relevant head-of-department > Relevant department senior >Chaplain > Regular crew

It may change in the future depending on how it plays out but this is what I currently have in mind.
## Why / Balance

For the orange alert, there never was a middle-ground between the blue and red alert which would result in confusion on which alert to pick during an attack of spiders, slimes or other critters. Both of the mentioned alerts don't state such threat, blue mentioning a confirmed threat and security that is now allowed to perform random searches while the red alert announces an active threat, something more fitting but still being too much for just a few hostile beings on the station.

As for the white alert, paranormal or unexplained activities have become more and more recent in the Umbra Sector, and so, I believe that there should be an alert level that is specifically there in case such events were to happen. During the alert, the RD and Chaplain would become more vital to the station as they would be the ones trying to counter or find a solution for the threat. This would be all mentioned in the altered Standard Operating Procedure in the SO A-2 subsection.
## Technical details

- Adds two new custom sound effects for the new alerts in a new _Umbra audio folder with their attributions.
- Adds an _Umbra alert-levels.ftl file in the Locale folder for the alerts.
- Adds the new alerts in the standard alert-levels.yml file.
## Media

orange-alert
orange-pda

white-alert
white-pda

- [x] I have added screenshots/videos to this PR showcasing its changes ingame, **or** this PR does not require an ingame showcase

## Breaking changes

**Changelog**

:cl:
- add: Added two new alert levels - orange and white. Orange alert is meant for hostile life forms while the white alert is meant for unexplained phenomena. Both have their own SOP entries.

### CODE REVIEW

File: Resources/Prototypes/AlertLevels/alert_levels.yml [hunk 1]
```diff
@@ -28,13 +28,27 @@
       emergencyLightColor: Goldenrod
       forceEnableEmergencyLights: true
       shuttleTime: 600
+    orange:
```
Comment (Simyon264) [inline Resources/Prototypes/AlertLevels/alert_levels.yml:31]: While the audio path makes it clear that this is an Umbra change, I would still put a comment there explaining that it is from Umbra.
```suggestion
    orange: # Sector-Umbra new alert levels.
```

File: Resources/Prototypes/AlertLevels/alert_levels.yml [hunk 2]
```diff
@@ -28,13 +28,27 @@
...
+      forceEnableEmergencyLights: true
+      shuttleTime: 600
     red:
       announcement: alert-level-red-announcement
       sound: /Audio/Misc/redalert.ogg
       color: Red
       emergencyLightColor: Red
       forceEnableEmergencyLights: true
       shuttleTime: 600 #No reduction in time as we don't have swiping for red alert like in /tg/. Shuttle times are intended to create friction, so having a way to brainlessly bypass that would be dumb.
+    white:
```
Comment (Simyon264) [inline Resources/Prototypes/AlertLevels/alert_levels.yml:45]: Same here.
```suggestion
    white:  # Sector-Umbra new alert levels.
```

File: Resources/Locale/en-US/_Umbra/alert-levels/alert-levels.ftl [hunk 3]
```diff
@@ -0,0 +1,7 @@
+alert-level-orange = Orange
+alert-level-orange-announcement = There are currently several unknown or hostile life forms aboard the station. All crew members are advised to remain in their departments and to report any aggresive entities to security.
+alert-level-orange-instructions = Crewmembers are advised to remain vigilant and report hostile life forms to security.
+
+alert-level-white = White
+alert-level-white-announcement = There is currently an unexplained abnormality on the station. All crewmembers are to remain vigilant, seek shelter, listen to command orders and to report any unknown phenomena to security or science.
```
Comment (Simyon264) [inline Resources/Locale/en-US/_Umbra/alert-levels/alert-levels.ftl:6]:
```suggestion
alert-level-white-announcement = There is currently an unexplained abnormality on the station. All crewmembers are to remain vigilant, seek shelter, listen to command orders and to report any unknown phenomena to security and science.
```

As the danger is not clear both departments being there makes more sense.

File: Resources/Locale/en-US/_Umbra/alert-levels/alert-levels.ftl [hunk 4]
```diff
@@ -0,0 +1,7 @@
...
+alert-level-white-instructions = Crewmembers are advised to remain in their departments and to report any unexplained or unnautral sightings to security or science.
```
Comment (Simyon264) [inline Resources/Locale/en-US/_Umbra/alert-levels/alert-levels.ftl:7]: Same as the above `and` instead of `or`

**Review by Simyon264 — CHANGES_REQUESTED** [review]

**Review by Simyon264 — APPROVED** [review]
Maintainer approval, still needs GM approval.

**Comment by TsjipTsjip** [comment]
GM approval. Wiki edit will be organised in sync with the next publish action.

**Pull request merged**